Assets/NpcTypes.cs

```diff
@@ -0,0 +1,11 @@
+namespace NpcSystem
+{
+    public static class NpcTypes 
+    {
+        public enum NpcType
+        {
+            Normal,
+            Armed
+        }
+    }
+}
```

Comment:
De enum kan zijn eigen file zijn, zonder een class.
```suggestion
namespace NpcSystem
{

        public enum NpcType
        {
            Normal,
            Armed
        }
}
```